src/viam/sdk/rpc/dial.cpp

```diff
@@ -6,12 +7,18 @@
 #include <boost/config.hpp>
 #include <boost/none.hpp>
 #include <boost/optional.hpp>
+#include <boost/utility/string_view.hpp>
 #include <grpcpp/channel.h>
+#include <grpcpp/create_channel.h>
 #include <grpcpp/security/credentials.h>
 
+#include <viam/api/proto/rpc/v1/auth.grpc.pb.h>
+#include <viam/api/proto/rpc/v1/auth.pb.h>
 #include <viam/api/robot/v1/robot.grpc.pb.h>
 #include <viam/api/robot/v1/robot.pb.h>
+#include <viam/sdk/common/client_helper.hpp>
 #include <viam/sdk/common/exception.hpp>
+#include <viam/sdk/common/private/instance.hpp>
 #include <viam/sdk/rpc/private/viam_grpc_channel.hpp>
 #include <viam/sdk/rpc/private/viam_rust_utils.h>
```

Comment:
(nit) Hm we did this wrong here (my bad!) but in most places in the SDK we've got the `api` includes in a separate block before the `sdk` includes. Could we do the same here?